nav_msgs/Path path
---
string resp
